moveit_ros/moveit_servo/src/servo_node.cpp

```diff
@@ -50,6 +50,15 @@
 namespace moveit_servo
 {
 
+rclcpp::Time convert_clock_type(const rclcpp::Time& time, rcl_clock_type_t new_clock_type)
```

Comment:
Clang-tidy didn't like my naming suggestion

```suggestion
rclcpp::Time convertClockType(const rclcpp::Time& time, rcl_clock_type_t new_clock_type)
```